app/src/testSpIP/java/org/dhis2/usescases/teiDashboard/ui/TeiDashboardMenuCustomActionsManagerImplTest.kt

```diff
@@ -0,0 +1,101 @@
+package org.dhis2.usescases.teiDashboard.ui
+
+import android.view.View
+import kotlinx.coroutines.Dispatchers
+import kotlinx.coroutines.test.runTest
+import org.dhis2.commons.viewmodel.DispatcherProvider
+import org.dhis2.usescases.sms.domain.model.sms.SmsResult
+import org.dhis2.usescases.sms.domain.usecase.SendSmsUseCase
+import org.dhis2.usescases.sms.presentation.contentprovider.SpipSmsContentResourcesProvider
+import org.dhis2.usescases.teiDashboard.TeiDashboardMenuCustomActionsManager
+import org.junit.Before
+import org.junit.Test
+import org.mockito.Mockito.verify
+import org.mockito.kotlin.doReturn
+import org.mockito.kotlin.mock
+import org.mockito.kotlin.never
+import org.mockito.kotlin.whenever
+
+class TeiDashboardMenuCustomActionsManagerImplTest {
```

Comment:
Please use conventions and helper functions used in EyeSeeTea as I've commented in the other test file